stock_picking_multiple_carrier/stock_picking.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 initOS GmbH (<http://www.initos.com>).
+#    Author Rami Alwafaie <rami.alwafaie at initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class StockPickingDelivery(models.Model):
+
+    _name = 'stock.picking.delivery'
+
+    carrier_id = fields.Many2one("delivery.carrier",
```

Comment:
Pass the arguments to the next line with an extra indentation level and you won't be too constrained by the remaining spaces